tests/tasks/webhook-gitlab/pullrequest-closed.yaml

```diff
@@ -7,5 +7,4 @@
/test","default_branch":"master","ci_config_path":null,"homepage":"https://gitlab.com/michael.schmid1/test","url":"git@gitlab.com:michael.schmid1/test.git","ssh_url":"git@gitlab.com:michael.schmid1/test.git","http_url":"https://gitlab.com/michael.schmid1/test.git"},"object_attributes":{"assignee_id":null,"author_id":2127130,"created_at":"2018-03-19 00:01:08 UTC","description":"bla bla bbla","head_pipeline_id":null,"id":"{{ git_pr_number }}","iid":1,"last_edited_at":"2018-03-19 00:04:33 UTC","last_edited_by_id":2127130,"merge_commit_sha":null,"merge_error":null,"merge_params":{"force_remove_source_branch":"0"},"merge_status":"unchecked","merge_user_id":null,"merge_when_pipeline_succeeds":false,"milestone_id":null,"source_branch":"{{ git_pr_branch }}","source_project_id":5796982,"state":"merged","target_branch":"{{ git_base_branch }}","target_project_id":5796982,"time_estimate":0,"title":"My Merge Request edit","updated_at":"2018-03-19 00:07:53 UTC","updated_by_id":2127130,"url":"https://gitlab.com/michael.schmid1/test/merge_requests/1","source":{"id":5796982,"name":"test","description":"","web_url":"https://gitlab.com/michael.schmid1/test","avatar_url":null,"git_ssh_url":"git@gitlab.com:michael.schmid1/test.git","git_http_url":"https://gitlab.com/michael.schmid1/test.git","namespace":"michael.schmid1","visibility_level":0,"path_with_namespace":"michael.schmid1/test","default_branch":"master","ci_config_path":null,"homepage":"https://gitlab.com/michael.schmid1/test","url":"git@gitlab.com:michael.schmid1/test.git","ssh_url":"git@gitlab.com:michael.schmid1/test.git","http_url":"https://gitlab.com/michael.schmid1/test.git"},"target":{"id":5796982,"name":"test","description":"","web_url":"https://gitlab.com/michael.schmid1/test","avatar_url":null,"git_ssh_url":"git@gitlab.com:michael.schmid1/test.git","git_http_url":"https://gitlab.com/michael.schmid1/test.git","namespace":"michael.schmid1","visibility_level":0,"path_with_namespace":"michael.schmid1/test","default_branch":"master","ci_config_path":null,"homepage":"https://gitlab.com/michael.schmid1/test","url":"git@gitlab.com:michael.schmid1/test.git","ssh_url":"git@gitlab.com:michael.schmid1/test.git","http_url":"https://gitlab.com/michael.schmid1/test.git"},"last_commit":{"id":"{{ git_pr_commit_hash }}","message":"edit test2","timestamp":"2018-03-19T00:06:27+00:00","url":"https://gitlab.com/michael.schmid1/test/commit/3f116b9ce232c1c84ddac414abb73e985b67aee2","author":{"name":"Michael Schmid","email":"michael.schmid@amazee.com"}},"work_in_progress":false,"total_time_spent":0,"human_total_time_spent":null,"human_time_estimate":null,"action":"close"},"labels":[],"changes":{"state":{"previous":"opened","current":"closed"},"updated_at":{"previous":"2018-03-19 00:06:28 UTC","current":"2018-03-19 00:07:53 UTC"},"total_time_spent":{"previous":null,"current":0}},"repository":{"name":"test","url":"git@gitlab.com:michael.schmid1/test.git","description":"","homepage":"https://gitlab.com/michael.schmid1/test"}}'
```

Comment:
this still says `action: close` was this maybe intended to also say `action: merge` ?